src/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form {
+  font-family: "Avenir", sans-serif;
```

Comment:
move font-family to the `html` tag selector and let inputs inherit it

Reply:
fixed.